.github/workflows/Bonsai.yml

```diff
@@ -0,0 +1,460 @@
+
+      # ----------------------------------------------------------------------- Pack
+      # Since packages are core to Bonsai functionality we always pack them even if they won't be collected
+      - name: Pack
+        id: pack
+        run: dotnet pack Bonsai.sln --no-restore --no-build --configuration ${{matrix.configuration}}
+
+      # ----------------------------------------------------------------------- Test
+      - name: Test
+        if: '!matrix.dummy-build'
+        run: dotnet test Bonsai.sln --no-restore --no-build --configuration ${{matrix.configuration}} --verbosity normal
+
+      # ----------------------------------------------------------------------- Create portable zip
+      - name: Create portable zip
+        id: create-portable-zip
+        if: matrix.create-installer
+        run: python .github/workflows/create-portable-zip.py artifacts/Bonsai.zip ${{matrix.configuration}}
+        env:
+          NUGET_API_URL: ${{vars.NUGET_API_URL}}
+          # This should be kept in sync with publish-packages-nuget-org
+          IS_FULL_RELEASE: ${{github.event_name == 'release' || (github.event_name == 'workflow_dispatch' && github.event.inputs.will_publish_packages == 'true' && github.event.inputs.version != '')}}
+
+      # ----------------------------------------------------------------------- Build Visual Studio templates
+      - name: Build Visual Studio templates
+        if: matrix.create-installer
+        run: msbuild /nologo /maxCpuCount Bonsai.Templates/Bonsai.Templates.sln /p:Configuration=${{matrix.configuration}} /p:DeployExtension=false
+
+      # ----------------------------------------------------------------------- Build setup
+      - name: Restore setup
+        if: matrix.create-installer
+        # Restoring the packages.config directly means we don't rely on the system-wide Wix install
+        run: |
+          nuget restore Bonsai.Setup/packages.config -SolutionDir .
+          nuget restore Bonsai.Setup.Bootstrapper/packages.config -SolutionDir .
+
+      - name: Build Setup
+        id: create-installer
+        if: matrix.create-installer
+        run: msbuild /nologo /maxCpuCount Bonsai.Setup.sln /p:Configuration=${{matrix.configuration}}
+      
+      # ----------------------------------------------------------------------- Collect artifacts
+      - name: Collect NuGet packages
+        uses: actions/upload-artifact@v4
+        if: matrix.collect-packages && steps.pack.outcome == 'success' && always()
+        with:
+          name: Packages${{matrix.artifacts-suffix}}
+          if-no-files-found: error
+          path: artifacts/package/${{matrix.configuration-lower}}/**
+
+      - name: Collect portable zip
+        uses: actions/upload-artifact@v4
+        if: steps.create-portable-zip.outcome == 'success' && always()
+        with:
+          name: PortableZip${{matrix.artifacts-suffix}}
+          if-no-files-found: error
+          path: artifacts/Bonsai.zip
```

Comment:
Not that I'm aware of. It's an ambiguous release artifact that has no relationship with the rest of the Bonsai build system, it could go anywhere really.

A lot of CI pipelines just emit this sort of stuff into the repo root since it'll evaporate into oblivion as soon as the CI run ends (but that feels a bit uncouth to me.)